### DISCUSSION TITLE
svg2rlg Inserts SVG Files into PDF but With Lots of Errors

### DISCUSSION THREAD

**Discussion by crabbylou** [deeplook/svglib#350] [Q&A] [answered]
Help. I have relied on svglib for years not to insert R graphs in SVG format into PDF files. 

I find now for some graphs that I get these errors, HOWEVER the graphs do insert, just lots of errors.

I am running version 1.1.0. I have not tried updating yet.

DEBUG:svglib.svglib:Exception during applyStyleOnShape
DEBUG:svglib.svglib:Exception during applyStyleOnShape
DEBUG:svglib.svglib:Exception during applyStyleOnShape
DEBUG:svglib.svglib:Unused attrs: path ['stroke', '__rules_applied']
DEBUG:svglib.svglib:Unused attrs: symbol ['overflow']
DEBUG:svglib.svglib:Unused attrs: use ['fill', 'fill-opacity', '__rules_applied', '{http://www.w3.org/1999/xlink}href']

(...over and over...)

It would not be so bad if the errors were just informational and I could somehow supress them.

My command in Python 3.6.8 is: drawing = svg2rlg(image_path).

Am I doing something wrong?

Any help appreciated.

Thanks,

Lou

**Comment by claudep** [comment, marked as answer]
The fact you find `DEBUG` in front of messages says it all: those are debug messages that shouldn't be displayed if the logging level is set to default (WARNING).
